api-extractor/src/definitions/ApiItem.ts

```diff
@@ -221,6 +227,22 @@ abstract class ApiItem {
     return true;
   }
 
+  public getJsDocs(): string {
+    if (!this.jsdocNode) {
+      return '';
+    }
+    const docComment: string = TypeScriptHelpers.getJsDocComments(this.jsdocNode, this.reportError);
```

Comment:
ApiItem can just do this when it's constructing the ApiDocumentation() instance.  If a unit test needs the raw string, we should just make that a property like ApiDocumentation.originalJsdoc.